src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/summaries/HotResourceSummary.java

```diff
@@ -223,14 +221,14 @@ public String getTableName() {
   public JsonElement toJson() {
     JsonObject summaryObj = new JsonObject();
     summaryObj.addProperty(SQL_SCHEMA_CONSTANTS.RESOURCE_TYPE_COL_NAME,
-        ResourceTypeUtil.getResourceTypeName(this.resourceType));
+        ResourceUtil.getResourceTypeName(this.resource));
+    summaryObj.addProperty(SQL_SCHEMA_CONSTANTS.RESOURCE_METRIC_COL_NAME,
+        ResourceUtil.getResourceTypeUnit(this.resource));
```

Comment:
There is a disconnect between column name and the getter function name. We should rename one of the two.